lib/host/android_notifications.dart

```diff
@@ -25,3 +25,29 @@ abstract class PendingIntentFlag {
   /// Corresponds to `FLAG_ALLOW_UNSAFE_IMPLICIT_INTENT`.
   static const allowUnsafeImplicitIntent = 1 << 24;
 }
+
+/// For use in [NotificationChannel.importance].
+///
+/// See: https://developer.android.com/reference/android/app/NotificationChannel#setImportance(int)
+abstract class NotificationImportance {
+  /// Corresponds to [`IMPORTANCE_UNSPECIFIED`](https://developer.android.com/reference/androidx/core/app/NotificationManagerCompat#IMPORTANCE_UNSPECIFIED())
```

Comment:
The `[text](example.com)` syntax makes it a bit harder to work out what the intended URL is and select it, especially here where the intended URL ends with ")".

I did notice Android Studio helpfully trying to linkify this in a hover interaction:

<img width="388" alt="image" src="https://github.com/user-attachments/assets/c213a260-ba7a-4439-92f4-19e8a98f19ed">

but then it similarly gets confused about what the intended URL is; it takes me here: https://developer.android.com/reference/androidx/core/app/NotificationManagerCompat#IMPORTANCE_UNSPECIFIED(

when it should have taken me here: https://developer.android.com/reference/androidx/core/app/NotificationManagerCompat#IMPORTANCE_UNSPECIFIED()

so the browser just leaves me at the top of the page, instead of scrolling me down to the `#IMPORTANCE_UNSPECIFIED()` element.

So I think probably most helpful to give up that syntax and just write some thing like:

```dart
  /// Corresponds to `IMPORTANCE_UNSPECIFIED`:
  ///   https://developer.android.com/reference/androidx/core/app/NotificationManagerCompat#IMPORTANCE_UNSPECIFIED()
```

(here and elsewhere in the PR)